### ISSUE TITLE
See the filesystem together with the Sas Content

### ISSUE THREAD

**Issue by gygabyte017** [sassoftware/vscode-sas-extension#417]
**Is your feature request related to a problem? Please describe.**
I need to access some sas file that are within the filesystem instead of the Sas Content

**Describe the solution you'd like**
See in the EXPLORER the filesystem together with the Sas Content, as Sas Studio does.

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Environment**
SAS version Viya 3.5

**Comment by scnwwu** [comment]
VS Code by itself can connect to a remote location and show remote file system in it's Explorer. Refers to https://code.visualstudio.com/docs/remote/remote-overview
Will it meet your need?

**Comment by gygabyte017** [comment]
@scnwwu Unfortunately I don't think so, because the file system shown in SAS Studio alongside the SAS Content is very hard to be connected to being on some volume in the kubernetes cluster in Viya 4. I don't think there is a secure and easy way to convince the Sas Cloud cysec to directly expose that internal volume on ssh in the network.

**Comment by clangsmith** [comment]
Ben Gow (a customer) wanted to make sure that when we add this feature that it also honors the fileNavigationCustomRootPath and serverDisplayName config properties, same as SAS Studio. I agree, this makes sense, so that there is a consistent experience between the extension and SAS Studio. So, please make sure to consider this in the implementation.

I am debating whether it makes sense to have separate config properties with the same names, but specific to the extension, or whether the extension should just re-use the existing SAS Studio config properties. The former would allow admins to set the properties separately for the extension and SAS Studio if they wanted them to be different for some reason. However, that would also force them to set both sets of properties if they wanted them to be consistent in the extension and SAS Studio (unless one overrode the other). For this reason, I think the latter makes more sense for now... re-use the SAS Studio config properties, so the extension and SAS Studio are consistent by default. If we get a request later, we could consider adding extension-specific properties that (if set) would override the SAS Studio config properties.

**Comment by jatran910** [comment]
I agree that this would be a good feature for the extension. It can help with determining whether issues are isolated or not to SAS Studio when working with the filesystem or configured NFS mounts by having another application that can also access those locations.

**Comment by clangsmith** [comment]
This should likely be done in coordination with the same feature request for V9 (#889), since they should have a largely consistent experience.

**Comment by scottdover** [comment]
This is currently a work in progress (see https://github.com/sassoftware/vscode-sas-extension/pull/1203). Since this work is pretty dense, we've decided to split things up in the following way:
 - Implement SAS file system for viya
 - Implement SAS file system for ITC-based connections (IOM/COM)
 - Implement favorites SAS file system with viya connections

Additionally, a bug was found when working on this feature set around downloading a mix of files/folders. A separate PR will be created to resolve that.

**Comment by snlwih** [comment]
Delivered as part of #1203 in [milestone 1.12](https://github.com/sassoftware/vscode-sas-extension/milestone/27)
